Sources/GameCore/GameCore.swift

```diff
@@ -17,16 +17,48 @@ import TcaHelpers
 import UpgradeInterstitialFeature
 
 public struct Game: Reducer {
+  public struct Destination: Reducer {
+    public enum State: Equatable {
+      case alert(AlertState<Action.Alert>)
+      case bottomMenu(BottomMenuState<Action.BottomMenu>)
+      case gameOver(GameOver.State)
+      case upgradeInterstitial(UpgradeInterstitial.State = .init())
+    }
+    public enum Action: Equatable {
+      case alert(Alert)
+      case bottomMenu(BottomMenu)
+      case gameOver(GameOver.Action)
+      case upgradeInterstitial(UpgradeInterstitial.Action)
+
+      public enum Alert: Equatable {
+        case forfeitButtonTapped
+      }
+      public enum BottomMenu: Equatable {
+        case confirmRemoveCube(LatticePoint)
+        case endGameButtonTapped
+        case exitButtonTapped
+        case forfeitGameButtonTapped
+        case settingsButtonTapped
+      }
+    }
+    public var body: some ReducerOf<Self> {
+      Scope(state: /State.gameOver, action: /Action.gameOver) {
+        GameOver()
+      }
+      Scope(state: /State.upgradeInterstitial, action: /Action.upgradeInterstitial) {
+        UpgradeInterstitial()
+      }
+    }
+  }
```

Comment:
`Game` and `Home` have some of the more interesting navigation refactors. Here we're able to house 4 destinations.